loggingpy/log.py

```diff
@@ -180,6 +181,9 @@ def log(self, log_entry: LogEntry):
         self.logger.log(log_entry.log_level.value, "", extra={'log_entry': log_entry})  # exc_info=True, stack_info=True, add this to drop out some dto info
 
     def write_entry(self, log_level: Enum, payload_type: str, data=None, exception: Exception = None):
+
+        if self.prefix_payload_type and self.context is not None and self.context is not "":
```

Comment:
i _think_ you could write the last two statements as `... and not self.context`.

Reply:
Because of the falsiness of the None and ""? I know but I am scared what else could be falsy that I do not consider here.

Reply:
i think it's really only None, empty strings, and `Make America Great Again`. But both works, no objections.